verification/VerifyTasks.wdl

```diff
@@ -331,42 +331,107 @@ task CompareLooms {
   }
 }
 
-task CompareH5adFiles {
+task CompareH5adFilesATAC {
   input {
     File truth_h5ad
     File test_h5ad
-
-    Int cpu = 1
-    String docker = "gcr.io/gcp-runtimes/ubuntu_16_0_4:latest"
-    Int disk_size_gb = ceil(size(truth_h5ad, "GiB") + size(test_h5ad, "GiB")) + 20
-    Int memory_mb = ceil(size(truth_h5ad, "MiB") + size(test_h5ad, "MiB") * 2)
+    String docker = "python:3.10.0-buster"
+    Int disk_size_gb = ceil(size(truth_h5ad, "GiB") + size(test_h5ad, "GiB")) + 200
+    Int memory_gb = 20
   }
 
   command <<<
 
     set -eo pipefail
 
-    # calculate file sizes
+    pip3 install anndata
+    
+    python3 <<CODE
+    
+    import anndata as ad
+    import numpy as np
+    import pandas as pd
+    
+    truth_h5ad = "~{truth_h5ad}"
+    test_h5ad = "~{test_h5ad}"
+    truth = ad.read_h5ad(truth_h5ad)
+    test = ad.read_h5ad(test_h5ad)
+    
+    truth_obs = pd.DataFrame(truth.obs)
+    test_obs = pd.DataFrame(test.obs)
+    
+    print("Now running obs equivalence check")
+    
+    if truth_obs.equals(test_obs)==True:
+        print("true")
```

Comment:
Could we print "pass" or a different message here? Is the "true" value used somewhere else?